test/configs/kv_accounts.conf

```diff
@@ -0,0 +1,23 @@
+port: -1
+net: localhost
+
+jetstream: enabled
+
+accounts: {
+    A: {
+        users: [ {user: a, password: a} ]
+        jetstream: enabled
+        exports: [
+            {service: '$JS.API.>' }
+            {service: '$KV.CROSS.>'}
```

Comment:
Does this support watchers also?

Reply:
This specific example probably not - but a watcher doesnt need any fiddling with subjects other than normal API related fiddling which of course is just up to if JS work or not. Ie. watch is up to ApiPrefix() passed to the JetStream context, nothing to do with the actual KV code.

Right? I think so anyway.

Reply:
Not sure, be good to flesh out a full example eventually, not needed here IMO. Thanks.

Reply:
Yeah need some stuff to make ordered consumer work cross account, not sure whats up - but yeah not for this PR its unrelated.